### PR TITLE
Improve new contributor readme, resolve missing conditional in Makefile

### PR DESCRIPTION
**Pull request by Firstyear**

### CODE REVIEW

**Comment by Firstyear** [comment]
cc @dmulder

File: README [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+## Contributing
+
+```bash
+sudo zypper install autoconf make gcc autogen gettext-tools
+```
+
+```bash
+cd pam-config
+autoreconf -fiv
+automake --add-missing
```
Comment (thkukuk) [inline README:18]: Instead of autoreconf and automake you should call `./autogen.sh`
Or does this not work for you?

**Review by thkukuk — CHANGES_REQUESTED** [review]

File: README (hunk 1 above)
Comment (Firstyear) [reply README:18]: Ahhh, I'm used to other projects where that isn't present. I'll update this.

**Comment by Firstyear** [comment]
Updated.

**Review by thkukuk — APPROVED** [review]

**Pull request merged**